ooniprobe/Utility/TFTColor/UIColor+TFT.h

```diff
@@ -0,0 +1,61 @@
+//
+//  UIColor+TFT.h
+//  Color
+//
+//  Created by Burhanuddin Sunelwala on 11/18/14.
+//  Copyright (c) 2014 TFT. All rights reserved.
```

Comment:
This does not look like super open source

Reply:
it is. please read the NOTICE file in this branch

Reply:
Sure, but someone should tell the guy that that copyright header is such that his code is not opensource since it declares "All rights reserved".

Reply:
That is the standard header of XCode. this is the license of the component 
https://github.com/burhanuddin353/TFTColor/blob/master/LICENSE

Reply:
If this is an external component, why are we not including it as a Pod?

In any case if we are vendorizing it we should ideally vendorize it with a script and at the very least include the LICENSE file.

Reply:
Because the POD is in swift. 
I don't understand what you said, the license file is already in their repo and I included the link in the NOTICE file

Reply:
The fact that the header says `All rights reserved` with no mention to the license leads to an unclear situation where, technically, the repository is licensed under the license but that file is "all rights reserved". Which means that, probably, we can pull it from the repository (provided that we do not violate the license) but we cannot apply changes. (I also wonder whether we can link to the object file generated by this file, of which I am not sure). Then, of course, the developer just used the default header of Xcode, and most likely he wanted to have the code fully opensource, but still - from what I remember from Nexa - it's a bit unclear. We may want to ask him to change the headers or even better open a pull request to him.